cheaters/README.md

```diff
@@ -12,34 +12,33 @@
 
 * The Minecraft: Bedrock Edition community works on a very strict no-cheater
 policy.
-	- This stems from our rather lax proof rules, and the assumption that
-	speedrunning is an inherently trust based competition.
+	- This stems from our rather lax proof rules and the assumption that
+	speedrunning is an inherently trust-based competition.
 * Minor infractions or general suspicion around a runner and/or run may result
 in said runner being placed on our suspicious players list.
-	- The implication is that runs from said runner will be studied in far
-	more depth than the typical run.
-	- The runner may also have additional restrictions placed upon them such
-	as for example, the requirement to show all perspectives in cooperative
-	runs.
-* Serious offences will result in a permanent leaderboard ban, and the removal
+	- This implies that runs from said runner will be studied in far more
+	depth than the typical run.
+	- The runner may also have additional restrictions placed upon them, such
+	as the requirement to show all perspectives in cooperative runs.
+* Serious offences will result in a permanent leaderboard ban and the removal
 of all previous runs.
 * While we do not keep track of every speedrunner ever, if we do find that a
 player has cheated in a game outside of Bedrock Edition, action will be taken.
 	- Players caught cheating in a game outside of the Minecraft series are
-	generally placed on the suspicious players list, but allowed to submit
+	generally placed on the suspicious players list but allowed to submit
 	runs.
-	- Players caught cheating in other games within the Minecraft series
-	such as Java Edition and Legacy Edition will generally be banned from
-	our leaderboards.
+	- Players caught cheating in other games within the Minecraft series,
+	such as Java Edition and Legacy Edition, will generally be banned from our
```

Comment:
Nitpick, but it sounds bad to use "our" here. The leaderboards are "owned" by the community, not by whoever writes the rules
```suggestion
	such as Java Edition and Legacy Edition, will generally be banned from the
```